tests/fixtures/basic-js/.sauce/config.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1alpha
+kind: folio
+sauce:
+  region: us-west-1
+  metadata:
+    name: Playwright Hello World
+    tags:
+      - e2e
+      - release team
+      - other tag
+    build: Release $CI_COMMIT_SHORT_SHA
+docker:
+  image:
+    name: saucelabs/stt-playwright-jest-node
+    tag: latest
+folio:
+  param:
+    browserName: "firefox"
```

Comment:
@dpgraham I'd avoid this for now till we have priority defined and its implemented on saucectl. What do you think?

Reply:
@vrunoa yeah I'll update this